src/style.css

```diff
@@ -0,0 +1,73 @@
+font-face {
+  font-family: Avenir, serif;
+  src: url(fonts/Avenir-Book.ttf);
+}
+  
+* {
+  box-sizing: border-box;
+  font-family: Avenir, serif;
+}
```

Comment:
inputs don't inherit fonts
https://stackoverflow.com/questions/6080413/input-doesnt-inherit-the-font-from-body

Reply:
I have tried to fix all problems, there is no ideal result but I did my best, please check)